ExplorerPatcher/dllmain.c

```diff
@@ -2491,14 +2491,16 @@ INT64 Shell_TrayWndSubclassProc(
                 HWND hShellTray_Wnd = FindWindowExW(NULL, NULL, L"Shell_TrayWnd", NULL);
                 INT64* CTrayInstance = (BYTE*)(GetWindowLongPtrW(hShellTray_Wnd, 0)); // -> CTray
                 const unsigned int TRAYUI_OFFSET_IN_CTRAY = 110;
-                uintptr_t TrayUIInstance = *((INT64*)CTrayInstance + TRAYUI_OFFSET_IN_CTRAY) + 8;
+                uintptr_t TrayUIInstance = *((INT64*)CTrayInstance + TRAYUI_OFFSET_IN_CTRAY);
                 if (TrayUIInstance)
                 {
+                    TrayUIInstance += 8;

```

Comment:
But no, still on the first position. Then what does that `if` check above actually check?